src/lib/Libpython/shared_python_utils.c

```diff
@@ -50,6 +50,8 @@
  * @brief get_py_homepath
  * 	Find and return where python home is located
  *
+ * NOTE: Caller need to free allocated memory to the variable "homepath" manually after use
```

Comment:
it would suffice to say that "the caller must free the memory allocated by this function"